### ISSUE TITLE
Random failure in Unit.Evolution.Systems.Cce.Equations

### ISSUE THREAD

**Issue by wthrowe** [sxs-collaboration/spectre#1719]
```
Unit.Evolution.Systems.Cce.Equations
-------------------------------------------------------------------------------
/home/wthrowe/spectre/tests/Unit/Evolution/Systems/Cce/Test_Equations.cpp:155
...............................................................................

/home/wthrowe/spectre/tests/Unit/TestingFramework.hpp:162: FAILED:
  CHECK( a == appx(b) )
with expansion:
  0.8863897765 == Approx( 0.8863897765 )
with messages:
  seed: 1541548361
  function: integrand_for_h_regular_part
  /home/wthrowe/spectre/tests/Unit/Pypp/CheckWithRandomValues.hpp:229: result =
  = pypp::call<ResultType>( module_name, function_name, std::get<ArgumentIs>
  (args)..., forward_to_pypp<ResultType>(std::get<MemberArgsIs>(member_args),
  used_for_size)...)
  a: ((1.118711899752703757e+05,5.690230376904918376e+04),(-7.
  621893607281114100e+02,1.542170585506433326e+03),(8.863897764530719314e-01,-
  1.736870284057513345e+02),(1.091757819021319165e+03,2.572008403459964939e+03)
  ,(-2.091211150003697912e+04,1.027834789986232681e+04))
  b: ((1.118711899752703757e+05,5.690230376904918376e+04),(-7.
  621893607281109553e+02,1.542170585506433326e+03),(8.863897764571664339e-01,-
  1.736870284057502261e+02),(1.091757819021318483e+03,2.572008403459964939e+03)
  ,(-2.091211150003697185e+04,1.027834789986232317e+04))
  a: 8.863897764530719314e-01
  b: 8.863897764571664339e-01
```

**Comment by nilsvu** [comment]
```
416/1019 Test #413: Unit.Evolution.Systems.Cce.Equations ........................................***Exception: Child aborted  0.90 sec
3078Charm++: standalone mode (not using charmrun)
3079Charm++> Running in Multicore mode:  1 threads
3080Converse/Charm++ Commit ID: v6.8.0-0-ga36028edb
3081Warning> Randomization of virtual memory (ASLR) is turned on in the kernel, thread migration may not work! Run 'echo 0 > /proc/sys/kernel/randomize_va_space' as root to disable it, or try running with '+isomalloc_sync'.
3082CharmLB> Load balancer assumes all CPUs are same.
3083Charm++> Running on 1 unique compute nodes (2-way SMP).
3084Charm++> cpu topology info is gathered in 0.000 seconds.
3085SpECTRE Build Information:
3086Version:                      0.0.0
3087Compiled on host:             914e991635ad
3088Compiled in directory:        /work/build_Debug_gcc-7
3089Source directory is:          /work/spectre
3090Compiled on git branch:       HEAD
3091Compiled with git hash:       83c4ccc9af4e2680d9f781e41a62be1d34bc684c
3092Linked on:                    Thu Oct 17 18:02:39 2019
3093
3094~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
3095"Debug.Unit.Evolution.Systems.Cce.Equations" is a Catch v2.2.1 host application.
3096Run with -? for options
3097
3098-------------------------------------------------------------------------------
3099Unit.Evolution.Systems.Cce.Equations
3100-------------------------------------------------------------------------------
3101/work/spectre/tests/Unit/Evolution/Systems/Cce/Test_Equations.cpp:155
3102...............................................................................
3103
3104/work/spectre/tests/Unit/TestingFramework.hpp:162: FAILED:
3105  CHECK( a == appx(b) )
3106with expansion:
3107  -0.0833357264 == Approx( -0.0833357264 )
3108with messages:
3109  Seed is: 415932756 from /work/spectre/tests/Unit/Pypp/CheckWithRandomValues.
3110  hpp:356
3111  function: integrand_for_h_regular_part
3112  /work/spectre/tests/Unit/Pypp/CheckWithRandomValues.hpp:230: result == pypp::
3113  call<ResultType>( module_name, function_name, std::get<ArgumentIs>(args)...,
3114  forward_to_pypp<ResultType>(std::get<MemberArgsIs>(member_args),
3115  used_for_size)...)
3116  a: ((2.765747034178320973e+03,3.161794113407413079e+03),(7.
3117  874947347469417309e+02,-8.333572644147935193e-02),(3.951144933113366278e+04,
3118  2.709511471507503302e+04),(-8.133118074601070475e+03,4.357677405794215701e+
3119  03),(-5.931760297102620825e+03,2.736477673359457640e+03))
3120  b: ((2.765747034178321428e+03,3.161794113407412169e+03),(7.
3121  874947347469424130e+02,-8.333572644602771362e-02),(3.951144933113366278e+04,
3122  2.709511471507503302e+04),(-8.133118074601070475e+03,4.357677405794215701e+
3123  03),(-5.931760297102621735e+03,2.736477673359457640e+03))
3124  a: -8.333572644147935193e-02
3125  b: -8.333572644602771362e-02
3126
31270.111 s: Unit.Evolution.Systems.Cce.Equations
3128===============================================================================
3129test cases:   1 |   0 passed | 1 failed
3130assertions: 140 | 139 passed | 1 failed
```

**Comment by moxcodes** [comment]
also closing because fix #1750 was merged.
